governance/parameter-docs/param-dai-savings-rate.md

```diff
@@ -9,45 +9,44 @@ Technical Docs: https://docs.makerdao.com/mcd-developer-guides/developer-guides-
 ```
 
 ## Description
-The Dai Savings Rate allows Dai holders to deposit Dai into the contract and earn interest paid in Dai. Maker Governance can alter the rate at which interest accrues.
+The Dai Savings Rate is a tool that Maker Governance can use to adjust the interest rate paid to DAI holders who have deposited DAI into the DSR contract. Maker Governance directly controls the Dai Savings rate.
 
-The Dai Savings rate is usually expressed as a percentage and can be any value greater than 0%. Therefore, the DAI Savings rate cannot be negative.
-
-The Dai Savings Rate can also be turned off by Maker Governance - rendering the interest rate in effect to be 0%.
+The Dai Savings Rate is usually expressed as a percentage and can be set to any value. However, while Governance can set the Dai Savings Rate to a negative value, this will not affect the balances held within the DSR as they cannot decrease; this is because the `drip` function will revert if negative.
```

Comment:
While I agree this isn't as clear as it could be and have changed the wording based on your advice, I was of the understanding that we are aiming this documentation at Governance participants rather than the general public. At least that's what I think @LongForWisdom told me. Happy to be corrected if wrong